src/loaders/features/enabled-features.js

```diff
@@ -1,20 +1,16 @@
-import { getConfigurationValue, getRuntime } from '../config/config'
+import { asyncFeatures, syncFeatures } from './features'
+import { getConfigurationValue, getRuntime } from '../../common/config/config'
 
-const featureNames = [
-  "ajax",
-  "jserrors",
-  "metrics",
-  "page_action",
-  "page_view_event",
-  "page_view_timing",
-  "session_trace",
-  "spa"
-]
+const featureNames = [...asyncFeatures, ...syncFeatures]
 
 function isEnabled(name, agentIdentifier) {
   return getRuntime(agentIdentifier).disabled !== true && getConfigurationValue(agentIdentifier, `${name}.enabled`) !== false
```

Comment:
I see this in places but it may be a relic. I don't think the runtime obj's `disabled` property ever gets set anywhere.

Reply:
https://issues.newrelic.com/browse/NEWRELIC-6480